tests/containers/container_runtime.pm

```diff
@@ -27,25 +27,27 @@
 use Mojo::Base 'containers::basetest';
 use testapi;
 use utils;
+use registration;
```

Comment:
Why is this here?

Reply:
yes should be removed, was in the old podman.pm.
Solved